mach_dep.c

```diff
@@ -232,7 +232,7 @@ GC_INNER void GC_with_callee_saves_pushed(void (*fn)(ptr_t, void *),
 
 # if defined(HAVE_PUSH_REGS)
     GC_push_regs();
-# else
+# elif !defined(NO_CRT)
```

Comment:
Should we perhaps either define an empty `GC_push_regs` and define `HAVE_PUSH_REGS`? or move `NO_CRT` to an outer `#if`? I am not sure it makes sense as part of else